documentation/5_Organizational_Agreements.md

```diff
@@ -65,4 +65,162 @@ process is defined by the IDSA certification working group.
 All the details on the IDS Certification scheme, the trust and assurance
 levels for component certification and operational certification, the
 certification criteria, and the process to approve the evaluation
-facilities are provided in [Chapter 4](./4_Technical_Agreements.md).
+facilities are provided in [Chapter 2](./2_Guiding_Principles.md).
+
+## Running data space instances
+
+### Intra- and inter data space instance governance
+
+It must be recognized that the role of IDS and IDSA is to fulfill
+
+1. within the broader landscape of existing data sharing initiatives and
+2. the ambition of a federation of interoperable data spaces.
+
+This implies that IDSA considers its development and deployment initiatives
+in the broader context of these two areas:
+
+The striving for interoperability within a data space (intra) so that
+IDS can provide a gradual migration path within a data space instance or
+data sharing initiative and preparing for interoperability between
+multiple data space instances or data sharing initiatives (inter) to
+pave the way for the federation of interoperable data spaces - as
+pursued by the European data strategy.
+
+The [Data Governance Act](https://digital-strategy.ec.europa.eu/en/policies/data-governance-act) also recommends both an intra data space
+(domain) governance authority and an inter data space (central)
+governance authority.
+
+'The recently proposed [Data Governance Act](https://digital-strategy.ec.europa.eu/en/policies/data-governance-act) confirms the notion of
+a governance structure constituted by multiple entities. For European
+data spaces, it is recommended to have a (domain) governance authority
+for each data space and a central governance authority overseeing all
+aspects in connection with interoperability of data spaces, i.e., the
+de-facto 'soft infrastructure'. This central authority will interact
+with all data space specific authorities.'
+
+As noted above, IDSA and the main IDS-stakeholders play an important
+role by working together to ensure the development and deployment of
+data space instances in the broader context of introducing new or
+migrating/ developing existing data sharing initiatives, (intra data
+space interoperability), and embedding them in the European ambition of
+a federation of interoperable data spaces (inter data space
+interoperability).
+
+![Relationship of OpenDEI Building
+Blocks and data space instances](media/media/image18.png)
+
+This structure of a data space reflects the role of the IDS data space
+framework in relation to the broadly used data sharing agreement
```

Comment:
which "broadly used data sharing agreement framework" is referenced here? I'm not aware of one that would be aligned with the IDSA Rulebook.